.php_cs

```diff
@@ -0,0 +1,3 @@
+<?php
+// TO BE REMOVED.
```

Comment:
yep, the non-dist file should not be under version control. i think it should even be ignored with .gitignore